### PR TITLE
clean up minor issue, add theme switch

### PR DESCRIPTION
**Pull request by djhahe**
### Summary (Please recap what you changed or fixed)
- update readme
- add chart source
- update nft icon

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
Screen Shot 2021-12-04 at 11 56 59
Screen Shot 2021-12-04 at 11 56 51
-   [] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

**Comment by gaupoit** [comment]
Quick question. Should we change the popup to dark theme also?
Screen Shot 2021-12-06 at 17 31 06

File: client/src/components/Common/MiddleTruncatedText/MiddleTruncatedText.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			</div>
+		</OverlayTrigger>
+	);
+};
+
+MiddleTruncatedText.defaultProps = {
+	end: 5,
+};
+
+function getEndString(fullString, end) {
```
Comment (gaupoit) [inline client/src/components/Common/MiddleTruncatedText/MiddleTruncatedText.jsx:21]: Should we move this helper function to helpers folder?

File: client/src/components/Common/SideBar/index.jsx [hunk 2]
```diff
@@ -116,9 +149,13 @@ const SIDEBAR_ITEMS = [
 	},
 ];
 
-const SideBar = ({ title, modules }) => {
+const SideBar = ({ modules }) => {
+	const url = window.location.pathname;
+	const title = url.split('/')[1];
```
Comment (gaupoit) [inline client/src/components/Common/SideBar/index.jsx:154]: Should we handle the root URL (eg: casperdash.io)? We might return "dashboard" as default title.

**Comment by djhahe** [comment]
> Quick question. Should we change the popup to dark theme also? Screen Shot 2021-12-06 at 17 31 06

It's quite tricky for now, bc the modal is rendering out side of the main container. I'll try to improve later

**Review by gaupoit — APPROVED** [review]
LGTM

**Pull request merged**